hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/ReconDBDefinition.java

```diff
@@ -74,6 +74,16 @@ public ReconDBDefinition(String dbName) {
           NSSummary.class,
           new NSSummaryCodec());
 
+  // Container Replica History with bcsId tracking.
+  public static final DBColumnFamilyDefinition
+      <Long, ContainerReplicaHistoryList> REPLICA_HISTORY_V2 =
```

Comment:
So this means the existing replica history table is left asis. Do we not have to worry about lost history, if not should there be a cleanup state step on upgrade?

Reply:
ContainerReplicaHistory is not a must have data, and hence the plan is to skip migrating data from the old column family to the new. If there is a need for that, we can write a one time migration on startup.

Reply:
Ok, can you file a jira for this, thanks.